### ISSUE TITLE
[windows Terminal>Settins] Luminosity ratio of the links present on the terminal chat settings page '2.228:1', which is less then required (4.5:1).

### ISSUE THREAD

**Issue by Saurabh220304** [microsoft/terminal#19685]
### Windows Terminal version

1.25.3571.0

### Windows build number

Windows 11 Version 26H1 (OS Build 28000.1340)

### Steps to reproduce

**Repro Steps:**
1.Launch the Window terminal app.
2.Navigate to the 'New Open Tab' and select the 'Settings' menu item.
3.Settings page will open.
4.Now navigate to the 'Terminal chat' tab and select it.
5.Terminal chat page will open.
6.Now check the luminosity ratio of the ''MINFW64' text.
7.Observe the issue here.

**User Experience:**
Users with low vision or color perception difficulties may struggle to read the links text clearly, making it harder to understand or activate the control independently, if its contrast ratio is less than '4.5:1'.

**WCAG Reference Link:**
[1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum)

**Attachements:**
https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-heparihar_microsoft_com/IQCAbfA0Pw09QJiSVgn6-ILLAbCl32gyq9iWwzek8YOX-Bk?e=SLxADU

### Expected Behavior

Luminosity ratio for ''Links'' text should be greater than or equal to '4.5:1'.

### Actual Behavior

Luminosity ratio of the links present on the terminal chat settings page '2.228:1', which is less then required (4.5:1).

**Comment by carlos-zamora** [comment]
<img width="1794" height="540" alt="Image" src="https://github.com/user-attachments/assets/53716ffd-d2f6-47ed-a976-3fdd7dc8a1cd" />

From looking at the screenshot, an easier fix may be to update the background of the info box. I bet we're using the wrong resource or something. This looks like it's an issue specifically when the terminal is in light mode.

**Comment by carlos-zamora** [comment]
Fixed by #19711
